packages/frontend/src/components/DashboardTiles/TileBase/index.tsx

```diff
@@ -182,7 +182,7 @@ const TileBase = <T extends Dashboard['tiles'][number]>({
                             </ButtonsWrapper>
                         ) : null}
                     </HeaderContainer>
-                    <ChartContainer className="non-draggable sentry-block fs-block cohere-block">
```

Comment:
This is going to cause many conflicts with your other PR https://github.com/lightdash/lightdash/pull/7376